### PR TITLE
Improve coverage and implementation for ModuleInitialize

### PR DESCRIPTION
**Pull request by mvz**

### CODE REVIEW

File: spec/reek/smell_detectors/module_initialize_spec.rb [hunk 1]
```diff
@@ -16,6 +16,16 @@ def initialize; end
                            source:  'string')
   end
 
+  it 'does not report with for a method with a different name' do
```
Comment (troessner) [inline spec/reek/smell_detectors/module_initialize_spec.rb]: I guess that should be "does not report a method with a different name"

Comment (mvz) [inline spec/reek/smell_detectors/module_initialize_spec.rb]: Yes, or something like that :smile:.

**Comment by troessner** [comment]
@mvz one tiny change and this one is ready to go....

**Comment by mvz** [comment]
I already fixed that but GitHub doesn't tell you about it :disappointed:.

**Comment by troessner** [comment]
>> I already fixed that but GitHub doesn't tell you about it 😞.

Too bad. Do you think thats because of our "squash-and-force-push" workflow?

**Comment by mvz** [comment]
> Do you think thats because of our "squash-and-force-push" workflow?

I don't think so. It's probably hard for GitHub to tell if something was fixed or just changed. OTOH, if you just add commits, GitHub can at least tell you which commits you haven't looked at yet.

**Comment by troessner** [comment]
True, but i was expecting an email when you change a line i reviewed, that is fairly easy to tell....

**Pull request merged**